.github/workflows/pytest.yml

```diff
@@ -43,6 +43,7 @@ jobs:
           stack-version: 7.14.2
       - name: Test with pytest
         run: |
+            bash exportlocal.sh
```

Comment:
Interesting.  This line cements the `exportlocal.sh` script as **the** place to specify envvars for the pytest Github check.  And since it's a shell script, can be easily run locally as well. 

Part of me thinks we should specify these variables under the `env` tag so that they're more visible when viewing the pytest.yml file. This would decouple the CI environment from developers local environments.

Reply:
no, please do not do this. Use the pytest.ini file to define ant variables for pytest

Reply:
ok I will delete it